Deepgram/Clients/Agent/v2/Websocket/Client.cs

```diff
@@ -51,10 +51,6 @@ public Client(string? apiKey = null, IDeepgramClientOptions? options = null) : b
     public async Task<bool> Connect(SettingsSchema options, CancellationTokenSource? cancelToken = null, Dictionary<string, string>? addons = null,

```

Comment:
@naomi-lgbt  I ran the agent examples and they all run. just see my comment on the arbitrary key example where I'm getting an error.